docs/app/pages/components/components-sections/input-controls/input-dropdown/input-dropdown.component.ts

```diff
@@ -71,9 +72,13 @@ export class ComponentsInputDropdownComponent extends BaseDocumentationSection i
     setFilter(filter: string): void {
         this.filter = filter;
         this.filteredOptionList =
-          this.filter && (this.filter.length > 0) ?
-          this.optionList.filter(option => (this.index(option.name) > -1)) :
-          this.optionList;
+            this.filter && (this.filter.length > 0) ?
+                this.optionList.filter(option => (this.index(option.name) > -1)) :
+                this.optionList;
+    }
+
+    dropdownOpenChange(value: boolean) {
+        this.dropdownOpen = value;
```

Comment:
The value here is getting emitted twice (not sure if that is an issue), could be linked to what you were saying about losing focus firing it as well as a click event etc. 

When I click on the selected option in the drop down it emits the value as false but the drop down is still open. Talked to you about this earlier. May not be an issues but thought I would highlight it just encase.